test/notify.spec.js

```diff
@@ -45,33 +45,69 @@ describe('TEST ALL NOTIFICATION ENDPOINTS', () => {
   it('should get one notifications', (done) => {
     chai
       .request(app)
-      .get('/api/v1/notifications/8')
+      .get('/api/v1/notifications/2')
       .set('authorization', userToken)
       .end((err, res) => {
         expect(res.body.status).to.equal('success');
-        expect(res.body.data.user);
+        expect(res.body).to.have.property('notify');
+        expect(res.body.notify).to.have.property('receiverId');
+        expect(res.body.notify).to.have.property('actorId');
+        expect(res.body.notify).to.have.property('action');
+        expect(res.body.notify).to.have.property('notifiable');
+        expect(res.body.notify).to.have.property('notifiableId');
+        expect(res.body.notify).to.have.property('isRead');
+        expect(res.body.notify).to.have.property('createdAt');
+        expect(res.body.notify).to.have.property('updatedAt');
         done(err);
       });
   });
-  it('should mark one notifications as read ', (done) => {
+  it('should mark a notifications as read', (done) => {
     chai
       .request(app)
-      .put('/api/v1/notifications/7')
+      .put('/api/v1/notifications/4')
       .set('authorization', userToken)
+      .set('Accept', 'application/json')
+      .send({ readStatus: 'setAsRead' })
       .end((err, res) => {
         expect(res.body.status).to.equal('success');
-        expect(res.body.data.notifications);
+        expect(res.body.message).to.equal('Notification successfully marked as read');
+        done(err);
+      });
+  });
+  it('should not mark a notifications as read twice', (done) => {
+    chai
+      .request(app)
+      .put('/api/v1/notifications/4')
+      .set('authorization', userToken)
+      .set('Accept', 'application/json')
+      .send({ readStatus: 'setAsRead' })
+      .end((err, res) => {
+        expect(res.body.status).to.equal('error');
```

Comment:
It is equally very important to test the returned `status code`. Please, let's include that